docs/integrations/builtin/core-nodes/n8n-nodes-base.respondtowebhook.md

```diff
@@ -30,21 +30,28 @@ Configure the node behavior using these parameters.
 
 Choose what data to send in the webhook response.
 
+- **All Incoming Items**: respond with all the JSON items from the input.
+- **Binary**: respond with a binary file defined in **Response Data Source**.
 - **First Incoming Item**: respond with the first incoming item's JSON.
-- **Text**: respond with a text defined in the **Response Body** field.
-- **JSON**: respond with a JSON object defined in the **Response Body** field.
-- **Binary**: respond with a binary file defined in the **Response Data Source** field.
+- **JSON**: respond with a JSON object defined in **Response Body**.
 - **No Data**: no response payload.
+- **Redirect**: redirect to a URL set in **Redirect URL**.
+- **Text**: respond with text set in **Response Body**.
 
 ## Node options
 
 Select **Add Option** to view and set the options.
 
 - **Response Code**: set the [response code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status){:target=_blank .external-link} to use.
 - **Response Headers**: define response headers to send.
+- **Put Response in Field**: available when you respond with **All Incoming Items** or **First Incoming Item**. Set the field name for the field containing the response data.
 
 ## Return more than one data item
 
+/// note | Deprecated in 1.22.0
+n8n 1.22.3 added support for returning all data items using the **All Incoming Items** option. n8n recommends upgrading to the latest version of n8n, instead of using the workarounds described in this section.
```

Comment:
```suggestion
n8n 1.22.0 added support for returning all data items using the **All Incoming Items** option. n8n recommends upgrading to the latest version of n8n, instead of using the workarounds described in this section.
```